pkg/catalog/debugger.go

```diff
@@ -81,6 +81,7 @@ func (mc *MeshCatalog) ListMonitoredNamespaces() []string {
 	namespaces, err := mc.namespaceController.ListMonitoredNamespaces()
 
 	if err != nil {
+		log.Error().Err(errListingNamespaces).Msg("Failed to list monitored namespaces")
```

Comment:
If both the error object's message (`.Err()`) and the log message (`.Msg()`) are the same, it seems unnecessary, and the error log would duplicate this error message which isn't useful.
If the `namespaceController.ListMonitoredNamespaces()` is already logging the error, this log is unnecessary.

Reply:
Good call out, thank you @shashankram! Closing this issue in favor of the error already being logged in `namespaceController.ListMonitoredNamespaces()`